### PR TITLE
Adds phaser library and the game

### PR DESCRIPTION
**Pull request by jakewilson**
here

### CODE REVIEW

File: src/client/game/Entity.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+var Entity = function(game, health) {
+  this._game = game;
+  this._health = health;
+  this._sprite = null;
+  this._created = false; // TODO is this needed?
```
Comment (JonathanFrias) [inline src/client/game/Entity.js]: If it's not being currently used, it's probably not needed!

Comment (jakewilson) [inline src/client/game/Entity.js]: :+1:

File: src/client/game/BulletPool.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+var NUM_BULLETS = 10;
```
Comment (kevinchesser) [inline src/client/game/BulletPool.js:1]: I would think we would want the player to have unlimited ammo. Or is this the number of bullets that can be on the screen at a time?

Comment (jakewilson) [inline src/client/game/BulletPool.js:1]: Number of bullets on screen at a time. We only construct 10 bullets and re-use them when they go off screen so we're not constructing new objects every time the user fires.

Comment (kevinchesser) [inline src/client/game/BulletPool.js:1]: Alright I like that :+1: We wouldn't want that scenario with the ark trees to happen to us

**Pull request merged**